skype/skype-ps/skype/Grant-CsTeamsMeetingBrandingPolicy.md

```diff
@@ -0,0 +1,143 @@
+---
+external help file: Microsoft.Teams.Policy.Administration.Cmdlets.Core.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://learn.microsoft.com/powershell/module/skype/grant-csteamsmeetingbrandingpolicy
+schema: 2.0.0
+title: Grant-CsTeamsMeetingBrandingPolicy
+author: szymonkatraMSFT
+ms.author: szymonkatra
+ms.reviewer:
+manager: stanlythomas
+---
+
+# Grant-CsTeamsMeetingBrandingPolicy
+
+## SYNOPSIS
+Assigns a teams meeting branding policy at the per-user scope. The CsTeamsMeetingBrandingPolicy cmdlets enable administrators to control the appearance in meetings by defining custom background, logo and colors.
+
+## SYNTAX
+
+### GrantToUser
+```
+Grant-CsTeamsMeetingBrandingPolicy -Identity <String> [[-PolicyName] <String>] [<CommonParameters>]
+```
+
+### GrantToGroup
+```
+Grant-CsTeamsMeetingBrandingPolicy [[-PolicyName] <String>] [-Group] <String> -Rank <Int32>
+ [<CommonParameters>]
+```
+
+### GrantToTenant
+```
+Grant-CsTeamsMeetingBrandingPolicy [[-PolicyName] <String>] [-Global] [-Force] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Assigns a teams meeting branding policy at the per-user scope. The CsTeamsMeetingBrandingPolicy cmdlets enable administrators to control the appearance in meetings by defining custom background, logo and colors.
+
+## EXAMPLES
+
+### Assign TeamsMeetingBrandingPolicy to user
+```powershell
+PS C:\> Grant-CsTeamsMeetingBrandingPolicy -identity "testuser@testtenantdomain.onmicrosoft.com" -PolicyName "Policy Test"
+```
+
+In the example shown above, the command will assign TeamsMeetingBrandingPolicy with name `Policy Test` to user `testuser@testtenantdomain.onmicrosoft.com`.
+
+### Assign TeamsMeetingBrandingPolicy to group
+```powershell
+PS C:\> Grant-CsTeamsMeetingBrandingPolicy -Group Testgroupassignments@testtenantdomain.onmicrosoft.com -PolicyName "Policy Test" -Rank 1
+```
+
+In the example shown above, the command will assign TeamsMeetingBrandingPolicy with name `Policy Test` to group `Testgroupassignments@testtenantdomain.onmicrosoft.com`.
```

Comment:
Can we use Contoso here?

Reply:
Changed to alice@contoso.com and group@contoso.com